clients/src/main/java/org/apache/kafka/common/requests/FindCoordinatorResponse.java

```diff
@@ -107,6 +107,14 @@ public int throttleTimeMs() {
         return throttleTimeMs;
     }
 
+    public boolean hasError() {
+        return this.error != Errors.NONE;
+    }
+
+    public boolean retriableError() {
```

Comment:
Do we need this? Could we just check if the exception is an instance of `RetriableException`?